### PR TITLE
Adjust insufficient default bpf_jit_limit

### PR DESCRIPTION
**Pull request by rfan1**
https://progress.opensuse.org/issues/153748

VR:http://openqa.suse.de/tests/13269605

### CODE REVIEW

**Comment by rfan1** [comment]
@foursixnine Please don't merge this PR before  S:m:32068:317786 is released

**Comment by rfan1** [comment]
> @foursixnine Please don't merge this PR before S:m:32068:317786 is released :)

File: tests/sles-15-SP4/sysctl.robot [hunk 1]
```diff
@@ -373,7 +373,7 @@ Sysctl_net_core_bpf_jit_kallsyms
     [Documentation]    Set to 1 from 0 for debuging purpose, os not released yet
     Sysctl Check Param Int    net.core.bpf_jit_kallsyms    1
 Sysctl_net_core_bpf_jit_limit
-    Sysctl Check Param Int    net.core.bpf_jit_limit    264241152
+    Sysctl Check Param Int    net.core.bpf_jit_limit    528482304
```
Comment (foursixnine) [inline tests/sles-15-SP4/sysctl.robot:376]:
```suggestion
    [Documentation]    Adjust insufficient default bpf_jit_limit (SLE Micro 5.4/Elemental) bsc#1218234
    Sysctl Check Param Int    net.core.bpf_jit_limit    528482304
```

Same below

File: tests/sles-15-SP5/sysctl.robot [hunk 2]
```diff
@@ -373,7 +373,7 @@ Sysctl_net_core_bpf_jit_kallsyms
     [Documentation]    Set to 1 from 0 for debuging purpose, os not released yet
     Sysctl Check Param Int    net.core.bpf_jit_kallsyms    1
 Sysctl_net_core_bpf_jit_limit
-    Sysctl Check Param Int    net.core.bpf_jit_limit    264241152
+    Sysctl Check Param Int    net.core.bpf_jit_limit    528482304
```
Comment (foursixnine) [inline tests/sles-15-SP5/sysctl.robot:376]:
```suggestion
    [Documentation]    Adjust insufficient default bpf_jit_limit (SLE Micro 5.4/Elemental) bsc#1218234
    Sysctl Check Param Int    net.core.bpf_jit_limit    528482304
```

**Review by foursixnine — APPROVED** [review]

**Pull request merged**